src/app/productConfig/productConfigForm/productConfigForm.tpl.html

```diff
@@ -1,4 +1,11 @@
 <form novalidate name="$ctrl.itemConfigForm" ng-submit="$ctrl.saveGiftToCart()">
+  <div class="row">
+    <span class="u-floatRight" style="margin-top: -16px">
+      Secure
```

Comment:
Can we add this label to `app.config.js` and translate it like our other labels?